keyboards/cannonkeys/reverie/readme.md

```diff
@@ -0,0 +1,28 @@
+# Reverie PCB
+
+Reverie PCB from CannonKeys
+
+This is for both the solderable and hotswap versions of this PCB. The hotswap version does not support per-key LEDs and has reduced layout support (no ISO)
```

Comment:
This line implies its 2 different PCBs, so probably should be 
```
keyboards/cannonkeys/reverie/soldered/
keyboards/cannonkeys/reverie/hotswap/
```
This way it can better encapsulate the hardware differences.

Reply:
I took a stab at this - I think its ok now